components/PublishUpdateBtnWithData.js

```diff
@@ -77,6 +77,10 @@ const selectOptions = [
     label: <FormattedMessage id="Update.notify.hostedCollectiveAdmins" defaultMessage="Hosted collective's admins" />,
     value: 'COLLECTIVE_ADMINS',
   },
+  {
+    label: <FormattedMessage id="Update.notify.noOne" defaultMessage="No one" />,
```

Comment:
:ice_cream: Not a request change, but `id` are not mandatory for new strings
```suggestion
    label: <FormattedMessage defaultMessage="No one" />,
```